src/ResourceManager/Network/Commands.Network/help/AzureRM.Network.md

```diff
@@ -63,7 +63,7 @@ Adds a backend address pool configuration to a load balancer.
 Adds a front-end IP configuration to a load balancer.
 
 ### [Add-AzureRmLoadBalancerInboundNatPoolConfig](Add-AzureRmLoadBalancerInboundNatPoolConfig.md)
-Adds an inbound NAT pool configuration to a load balancer.
```

Comment:
ping - please add descriptions back that are missing.

Reply:
Auto generation removes these references every time I auto generate help. Seems some issue with the source repo. These are not my changes.

Reply:
Auto generation help is removing these entries and is adding entries that do not exist in the original repo and are not part of my PR. There is some issue with the help files in the original repo. Fixing these is beyond the scope of this PR.